problems/week3_SORT/연서/K번째수/solution.js

```diff
@@ -0,0 +1,30 @@
+function solution(array, commands) {
+  var answer = [];
+
+  // commands 중첩 배열 분리를 위한 반복문
+  for (let i = 0; i < commands.length; i++) {
```

Comment:
```suggestion
  commands.forEach((item)=>{});
```
자스에서 for보다 forEach나 map 쓰는게 더 좋다고 하드라구영? 바꿀 수 있는건 바꾸자!

Reply:
이거 승희 금잔디 조에서 답 받아오면 공유해줘 궁금해잉

Reply:
map은 새로운 배열을 할당해서 메모리 초과를 발생시킬 수도 있으니까,,! 나중에 메모리를 신경써야하는 문제에서는 forEach 를 사용하면 좋지 않을까오오